lib/serverengine/process_manager.rb

```diff
@@ -204,14 +219,31 @@ def tick(blocking_timeout=0)
         return nil
       end
 
-      ready_pipes, _, _ = IO.select(@rpipes.keys, nil, nil, blocking_timeout)
+      if ServerEngine.windows?
+        @rpipes.each{|r, m|
```

Comment:
That's because Win32::Pipe::Server doesn't cause EOFError even when a child process exits.
(https://github.com/fluent/serverengine/pull/26/files#diff-1e5e735f236537f81cca69fb746da179R221)

Then I think we should adapt to child process exits by Process.waitpid2 and if child process exits, we have to close pipe.